src/pages/IPFSPin.vue

```diff
@@ -152,11 +159,12 @@
 <script>
 import 'vue2-dropzone/dist/vue2Dropzone.min.css'
 import { mapState } from 'vuex'
-import { store, aggregates } from 'aleph-js'
+import { store, broadcast, aggregates, ethereum, solana } from 'aleph-js'
 import { format, copyToClipboard } from 'quasar'
 const { humanStorageSize } = format
 
 import IPFS from 'ipfs'
+const shajs = require('sha.js')
 const isIPFS = require('is-ipfs')
```

Comment:
Please stick to a style of imports ESModules OR CommonJS.

Reply:
Done